src/App.tsx

```diff
@@ -15,29 +16,43 @@ class App extends React.Component<any, any> {
       box1: 'cell',
       box2: 'organism',
       modelProperties: {
-        albino: true,
+        albino: false,
         working_tyr1: false,
-        working_myosin_5a: false,
+        working_myosin_5a: true,
         open_gates: false
       },
       addEnzyme: false,
-      activeAssay: 'nucleus',
+      activeAssay: 'none',
+      mode: 'normal',
       substanceLevels: {
```

Comment:
Does this seem like a reasonable way to be tracking the levels of each substance in different parts of the cell? Sometimes it's a pain to use nested state like this, so I wanted to double-check.